pkg/azurefile/controllerserver.go

```diff
@@ -152,6 +153,8 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			}
 		case storageEndpointSuffixField:
 			storageEndpointSuffix = v
+		case networkEndpointTypeField:
+			networkEndpointType = v
```

Comment:
`createPrivateEndpoint = strings.EqualsFold(privateEndPointType, v)`

Reply:
Should we keep it as it is? This will be more extensible in case there's another type of endpoint in future.

Reply:
make sense.
While for parameter value check, should always use `strings.EqualsFold` to make case insensitive.

Reply:
Changed to strings.EqualsFold